tests/unit/test-firefox/test.firefox.js

```diff
@@ -1,19 +1,21 @@
 /* @flow */
 import path from 'path';
-import {describe, it} from 'mocha';
-import {assert} from 'chai';
+
 import deepcopy from 'deepcopy';
 import sinon from 'sinon';
 import FirefoxProfile from 'firefox-profile';
+import {describe, it} from 'mocha';
+import {assert} from 'chai';
+import {fs} from 'mz';
+
 import * as firefox from '../../../src/firefox';
 import {onlyInstancesOf, UsageError, WebExtError} from '../../../src/errors';
-import {fs} from 'mz';
 import {withTempDir} from '../../../src/util/temp-dir';
 import {TCPConnectError, fixturePath, fake, makeSureItFails} from '../helpers';
 import {basicManifest, manifestWithoutApps} from '../test-util/test.manifest';
-import {defaultFirefoxEnv} from '../../../src/firefox/';
 import {RemoteFirefox} from '../../../src/firefox/remote';
 
+const defaultFirefoxEnv = firefox.defaultFirefoxEnv;
```

Comment:
I tried that, but it looks like there are no default imports in `firefox.js` hence it fails.